distribution/src/bin/elasticsearch

```diff
@@ -45,6 +50,14 @@ then
   fi
 fi
 
+if [[ $ENROLL_TO_CLUSTER = true ]]; then
+  ES_MAIN_CLASS=org.elasticsearch.xpack.security.cli.EnrollNodeToCluster \
+    ES_ADDITIONAL_SOURCES="x-pack-env;x-pack-security-env" \
+    ES_ADDITIONAL_CLASSPATH_DIRECTORIES=lib/tools/security-cli \
+    "`dirname "$0"`"/elasticsearch-cli \
```

Comment:
This nested quoting and backtick stuff is a bit awkward to grok. Also not sure if this will cause issues in certain shells. @pugnascotia can probably provide better guidance here.

Reply:
This is what we use in all of our CLI bash scripts. Happy to consider alternatives if any

Reply:
Ah, if this is the convention we're using then that's fine by me. My brain just seemed to stall when I read past this initially. It's bash, so yeah, there's likely no alternative that is any less confusing to decipher.

Reply:
Well, you can use `$(...)` instead of backticks, which is better for various reasons, but if we want to make that change then we should do it everywhere as a standalone change.